nouvelles-perspectives-en-sciences-sociales.csl

```diff
@@ -536,7 +538,7 @@
       </choose>
     </layout>
   </citation>
-  <bibliography delimiter-precedes-last="never" et-al-min="11" et-al-use-first="7" initialize="false" subsequent-author-substitute="———" hanging-indent="true">
+  <bibliography delimiter-precedes-last="never" et-al-min="11" et-al-use-first="7" initialize="false" subsequent-author-substitute="&#8212;&#8212;&#8212;" hanging-indent="true">
```

Comment:
approved

Reply:
This modification will not affect our proposition ? We need a bibliography for our journal as the sample citations that csl-bot sent us. The citations generated and based on our proposed changes.

Reply:
@lsaadaoui, same here. The various hyphen and dash characters are hard to tell apart, so we use character codes for en-dashes and em-dashes.

Reply:
Thanks for the clarifications